src/components/Tiles/TilesGrid.js

```diff
@@ -0,0 +1,24 @@
+import ApiProvider from '../../providers/ApiProvider';
+import { render } from '../../shared/dom';
+import { MovieTile } from './MovieTile';
+
+export const TilesGrid = ({ renderOn, movies }) => {
+    const template = `
+        <div class='tiles-grid-container'>
+            ${movies
+                .map((movie, idx) => {
+                    // const movie_details = await ApiProvider.getTitleDetails(
+                    //     movie.id
+                    // );
+                    return `
+                        <div id='tiles-grid-item-${idx}'>
```

Comment:
I think that we don't need this container, also we don't need to add the idx to class, we won't be getting them from the DOM or anything (btw. if you had that in mind, for future, you could use the movie.id instead of idx, much safer)